README.md

```diff
@@ -100,7 +100,13 @@ To run the project, execute the following command:
 
 ## 👥 Authors <a name="authors"></a>
 
-> Mention all of the collaborators of this project.
+> Mention all of the collaborators of this project..
```

Comment:
- [ ] On your readme, the "Install" section contains no instructions.  Kindly, add those instructions like:

```bash
bundle
rails db:create
rails db:migrate
rails db:seed
```

That way, your users will be able to run your project every time. 💪